Pano/Models/Theme.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Theme.swift
+//  Pano
+//
+//  Created by Serdar Onur KARADAĞ on 14.07.2023.
+//
+
+import Foundation
+
+enum Theme {
+  case LIGHT
+  case DARK
```

Comment:
These cases should be lowercased
```suggestion
  case light
  case dark
```